hack/timestamps.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+function timestamps::time_wrapper() {
+    while IFS= read -r line; do
+        printf "%(%T)T: %s\n" "-1" "$line"
+    done
+}
+
+if [[ "${BASH_SOURCE[0]}" -ef "$0" ]]; then
+    set -o pipefail
```

Comment:
better to use PIPESTATUS imo, less intrusive
https://github.com/kubevirt/kubevirt/pull/3640/files#r444693453

are we safe with all the stderr and so as Jed said ?

Reply:
> better to use PIPESTATUS imo, less intrusive
> https://github.com/kubevirt/kubevirt/pull/3640/files#r444693453

What is the added value? I do not understand what this solves or how you expect to use it.

> are we safe with all the stderr and so as Jed said ?

Not sure what you refer to. If you refer to the redirection of stderr to stdout that is has already been pointed out, so what are you asking here?

Reply:
> > better to use PIPESTATUS imo, less intrusive
> > https://github.com/kubevirt/kubevirt/pull/3640/files#r444693453
> 
> What is the added value? I do not understand what this solves or how you expect to use it.

unless pipefail is inherited (lets assume its not, i need to recall / research the exact case it is),
its safe here, because there is just one pipe, and the leftmost is what you actually want (assuming the right side never fails).
Still, just if you want to consider to use
set +e
...
the command
return ${PIPESTATUS[0]}

which will get the same results, without pipefail, which is little more intrusive i think, but again maybe its totally safe here.
You raised it there, so i looked on other alternatives and then thought on the current one as well.

> 
> > are we safe with all the stderr and so as Jed said ?
> 
> Not sure what you refer to. If you refer to the redirection of stderr to stdout that is has already been pointed out, so what are you asking here?

Yes after i wrote this i read again all deeply.
Generally speaking i am not fan of causing any change by this script in this PR, but up to you and the reviewers.